code/controllers/subsystem/non_firing/SSmapping.dm

```diff
@@ -356,5 +356,44 @@ SUBSYSTEM_DEF(mapping)
 
 	log_world("Ruin loader finished with [budget] left to spend.")
 
+GLOBAL_VAR_INIT(maint_all_access, FALSE)
+GLOBAL_VAR_INIT(station_all_access, FALSE)
+
```

Comment:
If we're moving these to the SSmapping file shouldnt we also scope these vars to the SS?